### PR TITLE
feat(better-moodle-language): added a setting to force better-moodles language #158

### PR DESCRIPTION
**Pull request by YorikHansen**
# Feature

* **better-moodle-language:** added a setting to force better-moodles language (as proposed in [#158](https://github.com/jxn-30/better-moodle/issues/158)) ([de93a5d](https://github.com/jxn-30/better-moodle/commit/de93a5d77da54d8ed1d20a5363ce9b330adbde57))

### CODE REVIEW

File: redesign.user.js [hunk 1]
```diff
@@ -1153,17 +1171,24 @@ const isDashboard =
 const MOODLE_LANG = document.documentElement.lang.toLowerCase();
 const DARK_MODE_SELECTOR = 'html[data-darkreader-scheme="dark"]';
 
+const BETTER_MOODLE_LANG = // This is ugly. There probably is a better way to do this
+
+        GM_getValue(getSettingKey('general.language'), 'auto') === 'auto' ?
+            MOODLE_LANG
+        :   GM_getValue(getSettingKey('general.language'), 'auto');
+
```
Comment (jxn-30) [inline redesign.user.js:1179]: could an [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) be a more beautiful solution?
smth like that:
```js
const BETTER_MOODLE_LANG = (() => {
    const savedLanguage = GM_getValue(getSettingKey('general.language'), 'auto');
    if (savedLanguage === 'auto') return MOODLE_LANG;
    return savedLanguage;
})();
```

Also I think I would prefer this next to `MOODLE_LANG` initialization (in between `MOODLE_LANG` and `DARK_MODE_SELECTOR`)

File: redesign.user.js [hunk 2]
```diff
@@ -2064,6 +2089,10 @@ const SETTINGS = [
     new BooleanSetting('general.speiseplan', false),
     new BooleanSetting('general.googlyEyes', true),
     new BooleanSetting('general.semesterzeiten', false),
+    new SelectSetting('general.language', 'auto', [
+        'auto',
+        ...Object.keys(TRANSLATIONS),
+    ]),
```
Comment (jxn-30) [inline redesign.user.js:2095]: put this to the top of general (immediately after `'general.updateNotification'`) please

**Review by jxn-30 — CHANGES_REQUESTED** [review]

File: redesign.user.js (hunk 2 above)
Comment (YorikHansen) [reply redesign.user.js:2095]: I think this looks a bit weird

Comment (jxn-30) [reply redesign.user.js]: That is true. would it maybe look better if the select was fullwidth?
I think it is important to have the language at top but it should also look good.

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**